lambda-function/src/app.ts

```diff
@@ -38,43 +38,57 @@ export const lambdaHandler = async (event: S3Event): Promise<Measure> => {
   };
   let madieMeasure: Measure = {} as Measure;
   let emailId = "";
+  let matMeasure = {} as MatMeasure;
   const mailService: MailService = new MailService();
+
   try {
     const { Body } = await s3Client.send(new GetObjectCommand(params));
     const bodyContents = await streamToString(Body as Readable);
-    const matMeasure: MatMeasure = JSON.parse(bodyContents);
-
-    logAndMail("--------MAT Measure Details-------");
-    logAndMail(`User: ${matMeasure.harpId}`);
-    emailId = matMeasure.emailId;
-    logAndMail(`Measure id: ${matMeasure?.manageMeasureDetailModel.id}`);
-    logAndMail(`Measure name: ${matMeasure?.manageMeasureDetailModel.measureName}`);
-    logAndMail(`Measure version: ${matMeasure?.manageMeasureDetailModel.versionNumber}`);
-    logAndMail(`Measure revisionNumber: ${matMeasure?.manageMeasureDetailModel.revisionNumber}`);
-    logAndMail(`Measure cqlLibraryName: ${matMeasure?.manageMeasureDetailModel.cqllibraryName}`);
-    logAndMail(`CMS ID: ${matMeasure?.manageMeasureDetailModel.eMeasureId}`);
-    logAndMail("Converting measure from MAT to MADiE format");
+    matMeasure = JSON.parse(bodyContents);
     madieMeasure = convertToMadieMeasure(matMeasure);
-    logAndMail("Transferring measure over to MADiE");
+    emailId = matMeasure.emailId;
+
     const response = await new MeasureServiceApi(MADiE_SERVICE_URL, MADiE_API_KEY).transferMeasureToMadie(
       madieMeasure,
       matMeasure.harpId,
     );
-    logAndMail(`Transferred Measure id: ${response.id}`);
-    emailMessage = `${emailMessage}\nYour measure transfer was successful.`;
-    console.log("Lambda execution completed...");
-    await mailService.sendMail(emailId, "Successfully imported the Measure", emailMessage);
+    if (response) {
+      /* Success email */
+      logAndMail("--------Measure Details-------");
```

Comment:
just thinking, would it be good if we introduce some templates like a handlebar or is it too much for this use case?

Reply:
I would think it might be a little too much for three possible hardcoded cases